src/pages/Encounters/tabs/overview/FormsDialog.tsx

```
@@ -149,7 +136,11 @@ export const FormDialog = ({
 
   return (
     <>
-      <div className="flex" onClick={() => setOpen(true)}>
+      <div
+        className="flex"
+        onClick={() => setOpen(true)}
+        data-shortcut-id="add-questionnaire"
+      >
```

Biome (2.4.6)

[error] Static Elements should not be interactive.

(lint/a11y/noStaticElementInteractions)